### ISSUE TITLE
Dark mode

### ISSUE THREAD

**Issue by azumukupoe** [clementine-player/Clementine#6138]
### Before posting

Please follow the steps below and check the boxes with [x] once you did the step.

- [ ] I checked the issue tracker for similar issues
- [ ] I checked the [changelog](https://github.com/clementine-player/Clementine/blob/master/Changelog) if the issue is already resolved
- [ ] I tried the latest Clementine build from [here](https://builds.clementine-player.org/)

### System information

Please provide information about your system and the version of Clementine used.

- Operating System:
- Clementine version: 

### Expected behaviour / actual behaviour
Basically, invert coloring of UI is what I want.

**Comment by ronjouch** [comment]
I read in [Arch wiki / Qt](https://wiki.archlinux.org/index.php/Qt#Qt_Style_Sheets) that

> An interesting way of customizing the look and feel of a Qt application is using Style Sheets, which are just simple CSS files. Using Style Sheets, one can modify the appearance of every widget in the application.
> 
> To run an application with a different style just execute:
> 
> ```
> $ qt_application -stylesheet style.qss
> ```
> 
> For more information on Qt Style Sheets see the [official documentation](https://doc.qt.io/qt-5/stylesheet-reference.html) or other [tutorials](http://thesmithfam.org/blog/2009/09/10/qt-stylesheets-tutorial/).

And after a bit of googling, I found https://github.com/ColinDuquesnoy/QDarkStyleSheet , _"The most complete dark stylesheet for Qt application (Qt4, Qt5, PySide, PySide2, PyQt4, PyQt5, QtPy, PyQtGraph, Qt.Py)"_.

So, I gave it a shot:

1. `git clone git@github.com:ColinDuquesnoy/QDarkStyleSheet.git`
2. `cd QDarkStyleSheet/qdarkstyle`

Then, `qt5ct -stylesheet style.qss` works great: respects my Qt theme and is dark. However,

- `clementine -stylesheet style.qss` errors with `clementine: invalid option -- 'e'`
- `clementine -- -stylesheet style.qss` seems to start, with normal logging happening, but never finishes initializing, and doesn't crash but starts looping on something and consuming one full CPU, until killed.

<details>
<summary>Logs</summary>

```
01:23:04 /o/Q/qdarkstyle (master|✔) clementine -- -stylesheet style.qss
01:23:04.547 INFO  main:307                         Clementine-qt5 1.4 rc1
01:23:04.808 DEBUG NetworkProxyFactory:49           Detected system proxy URLs: ("", "", "", "")
01:23:04.809 DEBUG WorkerPool<HandlerType>:281      Starting worker 0x7f25fdffa4d0 "/usr/bin/clementine-tagreader" "/tmp/clementine_1097216943"
01:23:04.815 DEBUG WorkerPool<HandlerType>:281      Starting worker 0x7f25fdffa4d0 "/usr/bin/clementine-tagreader" "/tmp/clementine_1266997442"
01:23:04.821 DEBUG WorkerPool<HandlerType>:281      Starting worker 0x7f25fdffa4d0 "/usr/bin/clementine-tagreader" "/tmp/clementine_228560998"
01:23:04.829 DEBUG WorkerPool<HandlerType>:281      Starting worker 0x7f25fdffa4d0 "/usr/bin/clementine-tagreader" "/tmp/clementine_860282461"01:23:04.856 INFO  main:48                          TagReader worker connecting to "/tmp/clementine_1097216943"
01:23:04.857 DEBUG WorkerPool<HandlerType>:301      Worker 0x7f25e8006a20 connected to "/tmp/clementine_1097216943"
01:23:04.857 INFO  main:48                          TagReader worker connecting to "/tmp/clementine_228560998"
01:23:04.858 DEBUG WorkerPool<HandlerType>:301      Worker 0x7f25e80052a0 connected to "/tmp/clementine_228560998"01:23:04.865 INFO  main:48                          TagReader worker connecting to "/tmp/clementine_1266997442"
01:23:04.866 DEBUG WorkerPool<HandlerType>:301      Worker 0x7f25e8007060 connected to "/tmp/clementine_1266997442"01:23:04.869 INFO  main:48                          TagReader worker connecting to "/tmp/clementine_860282461"01:23:04.870 DEBUG WorkerPool<HandlerType>:301      Worker 0x7f25e8009f30 connected to "/tmp/clementine_860282461"
01:23:05.103 DEBUG QxtGlobalShortcutBackend:32      registering
01:23:05.311 DEBUG MainWindow:228                   Starting
__logging_message__01:23:05.418 WARN  unknown                          QCssParser::parseColorValue: Specified color without alpha value but alpha given: 'rgb 200, 200, 200, 75%'
__logging_message__01:23:05.530 WARN  unknown                          QCssParser::parseColorValue: Specified color without alpha value but alpha given: 'rgb 200, 200, 200, 75%'
01:23:05.554 DEBUG CoverProviders:35                Registered cover provider "MusicBrainz"
01:23:05.555 DEBUG CoverProviders:35                Registered cover provider "Discogs"
01:23:05.555 DEBUG CoverProviders:35                Registered cover provider "last.fm"
01:23:05.635 DEBUG MainWindow:308                   Initialising player
01:23:05.666 DEBUG MainWindow:314                   Creating models
01:23:05.812 INFO  Player:639                       Registered URL handler for "classicalradio"
01:23:05.812 DEBUG InternetModel:133                Adding internet service: "ClassicalRadio"
01:23:05.813 INFO  Player:639                       Registered URL handler for "di"
01:23:05.813 DEBUG InternetModel:133                Adding internet service: "DigitallyImported"
01:23:05.854 DEBUG InternetModel:133                Adding internet service: "Icecast"
01:23:05.860 DEBUG InternetModel:133                Adding internet service: "Jamendo"
01:23:05.861 INFO  Player:639                       Registered URL handler for "jazzradio"
01:23:05.861 DEBUG InternetModel:133                Adding internet service: "JazzRadio"
01:23:05.867 INFO  Player:639                       Registered URL handler for "magnatune"
01:23:05.869 DEBUG InternetModel:133                Adding internet service: "Magnatune"
01:23:05.880 DEBUG InternetModel:133                Adding internet service: "Podcasts"
01:23:05.881 INFO  Player:639                       Registered URL handler for "rockradio"
01:23:05.881 DEBUG InternetModel:133                Adding internet service: "RockRadio"
01:23:05.883 DEBUG InternetModel:133                Adding internet service: "SavedRadio"
01:23:05.884 INFO  Player:639                       Registered URL handler for "radiotunes"
01:23:05.884 DEBUG InternetModel:133                Adding internet service: "RadioTunes"
01:23:05.885 INFO  Player:639                       Registered URL handler for "somafm"
01:23:05.885 DEBUG InternetModel:133                Adding internet service: "SomaFM"
01:23:05.886 INFO  Player:639                       Registered URL handler for "intergalacticfm"
01:23:05.886 DEBUG InternetModel:133                Adding internet service: "Intergalactic FM"
01:23:05.936 DEBUG SpotifyService:108               Spotify system blob path: "/usr/bin/clementine-spotifyblob"
01:23:05.936 DEBUG SpotifyService:109               Spotify local blob path: "/home/ronj/.config/Clementine/spotifyblob/version16-64bit/blob"
01:23:05.937 DEBUG InternetModel:133                Adding internet service: "Spotify"
01:23:05.938 INFO  Player:639                       Registered URL handler for "subsonic"
01:23:05.982 DEBUG InternetModel:133                Adding internet service: "Subsonic"
01:23:06.077 INFO  Player:639                       Registered URL handler for "box"
01:23:06.077 DEBUG InternetModel:133                Adding internet service: "Box"
01:23:06.102 INFO  Player:639                       Registered URL handler for "dropbox"
01:23:06.102 DEBUG InternetModel:133                Adding internet service: "Dropbox"
01:23:06.125 INFO  Player:639                       Registered URL handler for "googledrive"
01:23:06.126 DEBUG InternetModel:133                Adding internet service: "Google Drive"
01:23:06.148 INFO  Player:639                       Registered URL handler for "seafile"
01:23:06.149 DEBUG InternetModel:133                Adding internet service: "Seafile"
__logging_message__01:23:06.151 WARN  unknown                          libpng warning: iCCP: known incorrect sRGB profile
01:23:06.179 INFO  Player:639                       Registered URL handler for "skydrive"
01:23:06.179 DEBUG InternetModel:133                Adding internet service: "OneDrive"
01:23:06.188 DEBUG CddaLister:98                    No CD devices found
01:23:06.190 WARN  DeviceKitLister:55               Error enumerating DeviceKit-disks devices: "org.freedesktop.DBus.Error.ServiceUnknown" "The name org.freedesktop.UDisks was not provided by any .service files"
01:23:06.204 DEBUG MainWindow:332                   Creating UI
01:23:06.223 DEBUG logging:68                       _g_io_module_get_default: Found default implementation gvfs (GDaemonVfs) for ‘gio-vfs’
01:23:06.286 DEBUG MainWindow:881                   Creating equalizer
01:23:06.287 DEBUG MainWindow:910                   Creating now playing widget
01:23:06.333 DEBUG Playlist:173                     k_max_scrobble_point 240
01:23:06.337 DEBUG MainWindow:972                   Loading settings
01:23:06.338 INFO  NetworkRemote:83                 Network Remote deactivated
^C⏎
```

</details>

I'm running [clementine 1.4.0rc1 packaged by Arch](https://www.archlinux.org/packages/community/x86_64/clementine/), on qt 5.14.2-1.
